CONTRIBUTING.md

```diff
@@ -81,6 +81,30 @@ cherry-picked into `book-v2` branch.
 
 Check the CI job after to do the Pull Request and then, click on in the `Details` of `netlify/kubebuilder/deploy-preview`
 
+## Understanding the versions
+
+|   Name	|   Example	|  Description |
+|---	|---	|---	|
+|  PROJECT version |  `V1`,`V2`,`V3` | After the Kubebuilder v3 introduces the plugins architecture to [Extensible CLI and Scaffolding Plugins](https://github.com/kubernetes-sigs/kubebuilder/blob/master/designs/extensible-cli-and-scaffolding-plugins-phase-1.md), the PROJECT version represents the version of the PROJECT file layout which are generated in the projects built with it.  |
+|  Releases version | `v2.2.0`, `v2.3.0`, `v2.3.1` | Represents the release tags of this project. See the [releases](https://github.com/kubernetes-sigs/kubebuilder/releases) page. |
+|  Plugin Versions | `go.kubebuilder.io/v2.0.0` | Represents the code source of the Plugin used to scaffold the projects. |
+
+Note that, we just need to bump a new PROJECT version if a breaking or significant change be introduce in the PROJECT file. Otherwise, it not requires be changed. And then, see that Releases and Plugin versions following the [semver](https://semver.org/) definition.  
```

Comment:
```suggestion
Note that PROJECT version should only be bumped if a breaking change is introduced in the PROJECT file format itself.  Changes to the Go scaffolding or the KubeBuilder CLI *do not* affect the PROJECT version.

Similarly, the introduction of a new major version (`(X+1).0.0`) of the Go plugin might only lead to a new minor (`X.(Y+1).0`) release of KubeBuilder, since no breaking change is being made to the CLI itself.  It'd only be a breaking change to KubeBuilder if we remove support for an older version of the plugin.

For more information on how the release and plugin versions work, see the the [semver](https://semver.org/) documentation.
```

Reply:
```markdown
Note that in the case of the `v1` and `v2` PROJECT version, a corresponding plugin version is implied and constant -- `v1` implies the `go.kubebuilder.io/v1` plugin, and similarly for `v2`.  This is for legacy purposes -- no such implication is made with the `v3` PROJECT version.
```

Reply:
Done: